NEWS

```diff
@@ -1,3 +1,8 @@
+* Version 1.2.11 (unreleased)
+- Added Namespace support (#30). radcli now supports Namespace on 
+  Linux based systems. 
+
```

Comment:
no need to add it as 1.2.11. 1.2.10 is unreleased. You can add it under rc_send_server_ctx entry.